packages/treasury/src/runLoC.js

```diff
@@ -0,0 +1,90 @@
+// @ts-check
+import { AmountMath } from '@agoric/ertp';
+import { handleParamGovernance, ParamType } from '@agoric/governance';
+import { Far } from '@agoric/marshal';
+import {
+  assertIsRatio,
+  assertProposalShape,
+  ceilMultiplyBy,
+  floorMultiplyBy,
+} from '@agoric/zoe/src/contractSupport/index.js';
+
+const { details: X, quote: q } = assert;
+
+export const CreditTerms = {
+  CollateralPrice: 'CollateralPrice',
+  CollateralizationRate: 'CollateralizationRate',
+};
+
+/**
+ * @param { ContractFacet } zcf
+ * @param {{ feeMintAccess: FeeMintAccess }} privateArgs
+ */
+const start = async (zcf, { feeMintAccess }) => {
+  const { main: initialValue } = zcf.getTerms();
+
+  const {
+    makePublicFacet,
+    makeCreatorFacet,
+    getParamValue,
+  } = handleParamGovernance(zcf, harden(initialValue));
+  const getRatio = name => {
+    const x = getParamValue(name);
+    assertIsRatio(x);
+    return /** @type { Ratio } */ (x);
+  };
```

Comment:
Seems generally useful. I'm fine with generalizing and adding these into the governance pkg.

Is it already time to refactor `paramManager` so it splits out all the different types? Or should we add helper tools like this for a while longer?

Reply:
Now would be a good time for me, while it's swapped in. And before we dig the hole deeper by using this API in more contracts.